saleor/core/tests/test_core.py

```diff
@@ -156,6 +156,12 @@ def test_create_fake_order(db, monkeypatch, image, media_root, warehouse):
         pass
     assert Order.objects.all().count() == 2
 
+    how_many_preorder_orders = 1
+    for _ in random_data.create_preorder_orders(how_many_preorder_orders):
+        pass
+    assert Order.objects.all().count() == how_many + how_many_preorder_orders
```

Comment:
```suggestion
    assert Order.objects.count() == how_many + how_many_preorder_orders
```
`.all()` can be skipped